frontend/public/components/operators.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import {Firehose, HorizontalNav, PageHeading} from './utils';
+import { referenceForModel } from '../module/k8s';
+import {SubscriptionsPage} from './operator-lifecycle-manager/subscription';
+import {ClusterServiceVersionsPage} from './operator-lifecycle-manager/clusterserviceversion';
+import {InstallPlansPage} from './operator-lifecycle-manager/install-plan';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  SubscriptionModel,
+} from '../models';
+
+const pages = [{
+  href: '',
+  name: 'Cluster Service Versions',
+  component: ClusterServiceVersionsPage,
+}, {
+  href: 'subscriptions',
+  name: 'Subscriptions',
+  component: SubscriptionsPage,
+}, {
+  href: 'installplans',
+  name: 'Install Plans',
+  component: InstallPlansPage,
+}];
+
+export const OperatorsPage: React.SFC<OperatorsPageProps> = ({match}) => {
+  const namespace = _.get(match, 'params.ns');
+  const title = 'Operators';
```

Comment:
@serenamarie125 What do you think about `Operator Lifecycle Management` for this aggregated OLM view?